scripts/scalingo-deploy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
```

Comment:
Suggestion d'explication de ce qu'est ce script

```suggestion
This script is meant to be run locally, to push a locally-made build to a scalingo machine.
```